### PR TITLE
[MM-22051] Remove To/From JSON (final)

### PR DESCRIPTION
**Pull request by streamer45**
#### Summary

Removed To/From JSON for the remaining structs in `model` (see commits).

#### Ticket

https://mattermost.atlassian.net/browse/MM-28985

#### Release Note

```release-note
NONE
```

### CODE REVIEW

File: app/web_conn_test.go [hunk 1]
```diff
@@ -254,10 +254,13 @@ func TestWebConnDrainDeadQueue(t *testing.T) {
 			i := seqNum
 			for err == nil {
 				_, buf, err = conn.ReadMessage()
-				ev := model.WebSocketEventFromJson(bytes.NewReader(buf))
-				require.LessOrEqual(t, int(i), limit)
-				assert.Equal(t, i, ev.GetSequence())
-				i++
+				if err != nil && len(buf) > 0 {
```
Comment (streamer45) [inline app/web_conn_test.go:257]: Added this to make the test pass. Let me know if we should instead accept empty inputs.

Comment (agnivade) [reply app/web_conn_test.go:257]: Fine with me.

**Review by agnivade — APPROVED** [review]
I've looked at it briefly. But I cannot say I have given it a thorough look. :P

**Review by isacikgoz — APPROVED** [review]
Looks good 🍬

**Pull request merged**